version.txt

```diff
@@ -1 +1 @@
-0.80.65
+0.80.65-microspector-1.0.9
```

Comment:
0.80.66

Reply:
Yeah, this will change for sure, this was temporary for our internal usage. Since I thought it breaks sth, I didn't want to increase the patch only.  I will.